stdlib/ballerina-grpc/src/main/java/org/ballerinalang/net/grpc/MessageConstants.java

```diff
@@ -122,5 +122,16 @@
     public static final String BOOLEAN = "boolean";
     public static final String BLOB = "blob";
     
-    
+    //stub template builder constants
+    public static final String COMMA = ",";
+    public static final String INPUT_TYPE_NAME = "req";
+    public static final String RESULT_TYPE_NAME = "result";
+    public static final String IGNORE_CAST = "_";
+    public static final String INIT_EMPTY_STUB = "Empty req = {};";
```

Comment:
can we move this code to template

Reply:
Using Template we cannot handle if-else itself. So we have to handle them in java level and cannot move that to there.